### ISSUE TITLE
Orientation handling

### ISSUE THREAD

**Issue by KingIdee** [amitshekhariitbhu/Android-MVP-Sample-Application#1]
If I perform a network operation and I rotate the screen, it restarts the operation. Any cure to this?

**Comment by amitshekhariitbhu** [comment]
Check [this](https://github.com/MindorksOpenSource/android-mvvm-architecture) project for screen-orientation handling.

**Comment by KingIdee** [comment]
I meant for MVP not MVVM

**Comment by KingIdee** [comment]
Hi! I meant handling orientation in an MVP architecture.

**Comment by Sagar0-0** [comment]
Yes we can use loaders for performing network operation in background and if we rotate the device then it will not reload or redo the network operation. i can do this using AsyncTaskLoader
